meta-dts-distro/recipes-dts/dts/dts/dts

```diff
@@ -71,12 +65,17 @@ while : ; do
     echo "  3) Restore firmware from Dasharo HCL report"
   fi
   if [ -n "${DES_IS_LOGGED}" ]; then
-      echo "  4) Edit DES keys"
+      echo "  4) Edit your DES keys"
+      echo "     Logs key    : " ${CLOUDSEND_LOGS_URL}
+      echo "     Download key: " ${CLOUDSEND_DOWNLOAD_URL}
+      echo "     Password    : " ${CLOUDSEND_PASSWORD}
+  fi
```

Comment:
@Firminator this `fi` is messing up the script as we later use else to show `Load your DES keys` while DES is not logged. Could you please fix that by just removing it?